src/navigation/UserVocabularyStackNavigator.tsx

```diff
@@ -24,6 +25,11 @@ const UserVocabularyStackNavigator = (): JSX.Element | null => {
         component={UserVocabularyOverviewScreen}
         options={({ navigation }) => options(overview, navigation)}
       />
+      <Stack.Screen
+        name='UserVocabularyCreation'
```

Comment:
Since i would like to put every child screen of the `UserVocabularyOverviewScreen` in a subfolder as `user-vocabulary` i think its better to start with `UserVocabularyXY` so f.e. `UserVocabularyEditScreen`

Reply:
This not our logic how we structure the routes-folder. We should not change this without talking about it in the team. I would really like if we stick to the existing logic OR have subfolders for every BottomNavigationTab (but we will have exceptions here, as some screens can be in different tabs, e.g. VocabularyDetail).

Reply:
alright. i renamed the Screens to `OverviewUserVocabularyScreen` and `EditUserVocabularyScreen`